nb/documentation/interactions_with_geometry/interactions_with_geometry.nb

```diff
@@ -8,6 +8,27 @@ Geometry can be given a color`;
 
 Box(10, 10, 10).color('pink').view();
 
+md`
+---
+### eachIn
+The individual components of a piece of geometry can be interacted with using .eachIn(). 
+The function takes two functions as arguments. The first function argument is applied to each element. 
+The second function argument determines what is returned. The result is returned as an array.
+`;
+
+const aGroup = Group(
+  Box(10, 200, 10),
+  Box(10, 200, 10).z(20),
+  Box(10, 200, 10).z(40)
+).view();
+
+Group(
+  ...aGroup.eachIn(
+    (s) => s.rz(Math.random()),
+    (b) => b
```

Comment:
How does the second argument here work? I should add an example for that

Reply:
The first function should transform individual elements.
The second function should transform the elements as a group.

e.g., a.eachIn(color('blue'), Link)

Reply:
What does Link() do? Maybe I need to document that!